modules/programs/kitty.nix

```diff
@@ -57,6 +54,25 @@ let
     '';
   };
 in {
+  imports = [
+    (mkChangedOptionModule [ "programs" "kitty" "theme" ] [
+      "programs"
+      "kitty"
+      "themeFile"
+    ] (config:
+      let value = getAttrFromPath [ "programs" "kitty" "theme" ] config;
+      in if value != null then
+        (let
+          matching = filter (x: x.name == value) (builtins.fromJSON
+            (builtins.readFile
+              "${pkgs.kitty-themes/share/kitty-themes/themes.json}"));
+        in throwIf (length matching == 0)
+        "kitty-themes does not contain a theme named ${value}"
+        (head matching).file)
```

Comment:
I think you'll need to remove the `themes/` prefix and the `.conf` suffix from this value to be compatible with the expected format of `themeFile`.

Reply:
Glad you caught this. I ended up writing `tests/modules/programs/kitty/theme-to-themeFile.nix` to test this. 

There was also another mistake I made a couple lines up (curly brace is closed too late when I tried to refer to `pkgs.kitty-themes`) that the test caught.